### PR TITLE
Use BITOPS_PLUGINS_DIR to fix the plugin unable to find the deployment scripts

### PR DESCRIPTION
**Pull request by mickmcgrath13**
Clean up seemingly unused code (namely `scripts/deploy.sh`), and use the correct reference for bitops installed plugins (from https://github.com/bitovi/bitops/pull/347).

Error:
```
+ bash /opt/bitops/scripts/plugins/ansible/scripts/validate_env.sh
bash: /opt/bitops/scripts/plugins/ansible/scripts/validate_env.sh: No such file or directory
+ echo 'Running Ansible Playbooks'
Running Ansible Playbooks
+ bash -x /opt/bitops/scripts/plugins/ansible/scripts/ansible_install_playbooks.sh ''
bash: /opt/bitops/scripts/plugins/ansible/scripts/ansible_install_playbooks.sh: No such file or directory
```

### CODE REVIEW

**Review by armab — APPROVED** [review]

File: deploy.sh [hunk 1]
```diff
@@ -34,23 +34,13 @@ fi
...
-# source "$BITOPS_SCHEMA_ENV_FILE"
-
-
 # Check if the BITOPS_ANSIBLE_INVENTORY value is set
 if [ -n "$BITOPS_ANSIBLE_INVENTORY" ]; then
   BITOPS_CONFIG_COMMAND="$BITOPS_CONFIG_COMMAND --inventory-file=$BITOPS_ANSIBLE_INVENTORY"
 fi
 
-bash $SCRIPTS_DIR/plugins/ansible/scripts/validate_env.sh
+bash $BITOPS_PLUGINS_DIR/ansible/scripts/validate_env.sh
```
Comment (mickmcgrath13) [inline deploy.sh:42]: should be
```
bash $BITOPS_INSTALLED_PLUGIN_DIR/scripts/validate_env.sh
```

File: deploy.sh [hunk 2]
```diff
@@ -34,23 +34,13 @@ fi
...
 
 
 echo "Running Ansible Playbooks"
-bash -x $SCRIPTS_DIR/plugins/ansible/scripts/ansible_install_playbooks.sh "$BITOPS_CONFIG_COMMAND"
-
-# bash $SCRIPTS_DIR/deploy/after-deploy.sh "$ANSIBLE_ROOT"
+bash -x $BITOPS_PLUGINS_DIR/ansible/scripts/ansible_install_playbooks.sh "$BITOPS_CONFIG_COMMAND"
```
Comment (mickmcgrath13) [inline deploy.sh:46]: should be
```
bash -x $BITOPS_INSTALLED_PLUGIN_DIR/scripts/ansible_install_playbooks.sh "$BITOPS_CONFIG_COMMAND"
```

Comment (PhillypHenning) [reply deploy.sh]: Agreed

**Review by LeoDiazL — APPROVED** [review]
Double checked. Working OK.

**Pull request merged**